build/charts/antrea-windows/templates/daemonset.yaml

```diff
@@ -62,6 +62,10 @@ spec:
               - disable
         {{- end}}
         name: antrea-agent
+        resources:
+            requests:
+              cpu: 100m
+              memory: 100Mi
```

Comment:
are the memory requests based on some observations for a Windows Node running Antrea?